### DISCUSSION TITLE
Interest in a home-lab show and tell series?

### DISCUSSION THREAD

**Discussion by Mellowlynx** [tteck/Proxmox#3181] [General]
Hi all,

I'm wondering if there are people here that would like to share about their home-lab.
As I'm active here, I got some direct questions not long ago on why I run some parts of my lab as I do.
It gave the person a different way of looking at a situation, and together we made a system that works for him.

Now I was thinking to maybe start some sort of write-up series, in where someone can share about their lab.
What they run and why, something they wish to know earlier, some general advice, etc.

But why (here)?
Plenty of Proxmox users come here to get great software easily deployed within their lab, thanks to @tteck (BTW, support him if you use the scripts on the regular!) therefore, I think this is a great place/community to share stories and inspire others to learn and improve.

But no, it should probably not be added within the repository discussions, but it would be cool if it would be able to be connected to here in some way.  
This might be a good addition for this community, so I would like to know what others think.

It this something you would like to read and contribute too?
How and where do you think is a good way to share? Discord, Forum, Something else?
Do you have a similar or other idea that might be a good addition to this community?

Please don't hesitate to react, and maybe we can make something happen.

**Comment by di5cord20** [comment]
I would absolutely be a reader.  Not sure i have enough experience to contribute (yet).

My use case thus far has been replacing docker containers with LXCs.  It's a personal choice.  I like the ease of LXC backups and restores.  My next undertaking is deploying an Opnsense VM which I have yet to do.  Would be interested in reading other people's experiences.

**Reply by thecrlgh** [reply]
100%. I'm still very much learning. Would love to get some inspiration!

**Reply by Mellowlynx** [reply]
Thanks, both for your feedback.
